src/synthesizer/parametric/sf_hist.py

```diff
@@ -316,11 +293,16 @@ def _sfr(self, age):
             age (float)
                 The age (in years) at which to evaluate the SFR.
         """
-        if age < self.max_age:
+
+        if (age < self.max_age) and (age > self.min_age):
             return np.exp(-age / self.tau)
         return 0.0
 
 
+# included for backwards compatability
+TruncatedExponential = Exponential
```

Comment:
Let's just remove this truncated completely

Reply:
Don't remove it just yet but don't do this either. 

You should make a class that just calls `Exponential.__init__` in it's own `init`. You can then add the deprecated decorator to `TruncatedExponential.__init__` which will warn that it will be removed. 

e.g.
```
class TruncatedExponential(Exponential):
    @deprecated
    def __init__(self, *args, **kwargs):
        Exponential.__init__(self, *args, **kwargs)
```

Reply:
Yeah, done.

Reply:
Although... the same argument stands here. We could keep `TruncatedExponential` as a nice wrapper and have the max and min arguments as positional (i.e. they are needed for a truncated exponential SFH).

Reply:
Done.